weave/guides/platform/weave-self-managed.mdx

```diff
@@ -105,323 +83,611 @@ helm repo add altinity https://helm.altinity.com
-    
+                      name: aws-creds
+                      key: aws_secret_key
+
+              securityContext:
+                allowPrivilegeEscalation: false
+                capabilities:
+                  drop:
+                    - ALL
+                privileged: false
+                readOnlyRootFilesystem: false
+
     volumeClaimTemplates:
-      - name: data-volume-template
+      - name: data
+        metadata:
+          labels:
+            app: clickhouse-server
         spec:
           accessModes:
             - ReadWriteOnce
           resources:
             requests:
-              storage: 100Gi
-          # Update this to match your storage class
-          storageClassName: gp3
+              storage: 50Gi
+          storageClassName: gp3 # Change to your StorageClass
+
+  configuration:
+    # Keeper (ZooKeeper) configuration
+    # IMPORTANT: These hostnames MUST match your Keeper deployment from Step 3
+    zookeeper:
+      nodes:
+        - host: chk-wandb-keeper-0-0.clickhouse.svc.cluster.local
+          port: 2181
+        - host: chk-wandb-keeper-0-1.clickhouse.svc.cluster.local
+          port: 2181
+        - host: chk-wandb-keeper-0-2.clickhouse.svc.cluster.local
+          port: 2181
+      # Optional: Uncomment to adjust timeouts if needed
+      # session_timeout_ms: 30000
+      # operation_timeout_ms: 10000
+
+    # User configuration
+    users:
+      weave/password: weave123
+      weave/access_management: 1
+      weave/profile: default
+      weave/networks/ip:
+        - "0.0.0.0/0"
+        - "::"
+
+    # Server settings
+    settings:
+      disable_internal_dns_cache: 1
+
+    # Cluster configuration
+    clusters:
+      - name: weavecluster # Cluster name - can be customized but must match wandb-cr.yaml
+        layout:
+          shardsCount: 1
+          replicasCount: 3 # Number of replicas - adjust based on HA requirements
+        templates:
+          podTemplate: clickhouse
+          dataVolumeClaimTemplate: data
+
+    # Configuration files
+    files:
+      config.d/network_configuration.xml: |
+        <clickhouse>
+            <listen_host>0.0.0.0</listen_host>
+            <listen_host>::</listen_host>
+        </clickhouse>
+
+      config.d/logger.xml: |
+        <clickhouse>
+            <logger>
+                <level>information</level>
+            </logger>
+        </clickhouse>
+
+      config.d/storage_configuration.xml: |
+        <clickhouse>
+            <storage_configuration>
+                <disks>
+                    <s3_disk>
+                        <type>s3</type>
+                        <!-- Update with your S3 bucket endpoint and region -->
+                        <endpoint>https://YOUR-BUCKET-NAME.s3.YOUR-REGION.amazonaws.com/s3_disk/{replica}</endpoint>
+                        <metadata_path>/var/lib/clickhouse/disks/s3_disk/</metadata_path>
+                        <!-- Use environment credentials (IRSA) or remove for access keys -->
```

Comment:
This comment can be removed, since I created the credentials and added them to env vars, the ClickHouse will pull it from there, as if it was using IRSA.